homeassistant/components/lupusec/alarm_control_panel.py

```diff
@@ -39,6 +39,20 @@ def setup_platform(
     add_entities(alarm_devices)
 
 
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Set up an alarm control panel for a Lupusec device."""
+    data = hass.data[LUPUSEC_DOMAIN]
+
+    alarm_devices = [LupusecAlarm(data, data.lupusec.get_alarm(), config_entry)]
+
+    async_add_devices(alarm_devices)
+
+
+def get_unique_id(config_entry_id: str, key: str) -> str:
+    """Create a unique_id id for a lupusec entity."""
+    return f"{LUPUSEC_DOMAIN}_{config_entry_id}_{key}"
```

Comment:
I am leaning towards solving this in a follow up as well

Reply:
Removed.